web/yo/app/scripts/controllers/gene.js

```diff
@@ -2397,7 +2239,7 @@ angular.module('oncokbApp')
 
             $scope.getAllCitations = function() {
                 var results = [];
-                var geneData = JSON.stringify(stringUtils.getGeneData(this.gene, true, true, true));
+                var geneData = JSON.stringify(stringUtils.getGeneData(this.gene, true, false));
```

Comment:
Why we change from true to false?

Reply:
Previously, getGeneData() function took 5 parameters, which means the last parameter onlyReviewedContent is using the default false value.